### PR TITLE
Fix errors in debug building

### PR DESCRIPTION
**Pull request by PeterWangIntel**
This patch consists of two parts: devtools-related and net-related.
All of them are harmless for release version.

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@d0ngjun

File: net/log/net_log_util.cc [hunk 1]
```diff
@@ -203,11 +203,14 @@ scoped_ptr<base::DictionaryValue> GetNetConstants() {
   {
     base::DictionaryValue* dict = new base::DictionaryValue();
 
+// TODO (Peter Wang): Remove the temp fix for debug building.
```
Comment (d0ngjun) [inline net/log/net_log_util.cc]: Seems the code below is related with QUIC, you can try to use macro DISABLE_QUIC_SUPPORT to control it.

Comment (PeterWangIntel) [inline net/log/net_log_util.cc]: thx for the info, let me try

**Comment by d0ngjun** [comment]
LGTM

**Pull request merged**